identity/app/services/MembersDataApiService.scala

```diff
@@ -0,0 +1,80 @@
+package services
+
+import play.api.libs.ws.{DefaultWSCookie, WSClient, WSCookie, WSResponse}
+import play.api.mvc.{Cookie, Cookies}
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import utils.SafeLogging
+import scala.concurrent.{ExecutionContext, Future}
+
+// Modeled on members-data-api Attributes - ContentAccess
+// https://github.com/guardian/members-data-api/blob/master/membership-attribute-service/app/models/Attributes.scala
+case class ContentAccess(
+  isMember: Boolean,
+  isPaidMember: Boolean,
+  isRecurringContributor: Boolean,
+  hasDigitalPack: Boolean,
+  isPaperSubscriber: Boolean,
+  isGuardianWeeklySubscriber: Boolean
+) {
+  def canProceedWithAutoDeletion: Boolean = !(isMember || isPaidMember || isRecurringContributor || hasDigitalPack || isPaperSubscriber || isGuardianWeeklySubscriber)
+  def hasSubscription: Boolean = hasDigitalPack || isPaperSubscriber || isGuardianWeeklySubscriber
+}
+
+object ContentAccess {
+  implicit val jsRead: Reads[ContentAccess] = (
```

Comment:
This should not be necessary. Remove `is/has` prefixes from model fields and use `Json.format[ContentAccess]`